app/models/order.py

```diff
@@ -107,10 +107,7 @@ def invoice_number(self):
         return self.get_invoice_number()
 
     def get_tickets_count(self):
-        count = 0
-        for order_ticket in self.order_tickets:
-            count += order_ticket.quantity
-        return count
+        return sum([t.quantity for t in self.order_tickets])
```

Comment:
You will add the `tickets_count` property corresponding to this method, just like with the case of `invoice_number`, right?

Reply:
done :)